DEVELOPMENT.md

```diff
@@ -1,261 +0,0 @@
-# Development Workflow
-
-## Set Up
```

Comment:
Can we still keep some of the info in the root DEVELOPMENT.md? For example, the set up, code formatting, and running integration tests.

Reply:
Done. Yes, the pre-commit hooks and format should be at the root.